Formula/g/git-spice.rb

```diff
@@ -20,6 +20,8 @@ class GitSpice < Formula
 
   depends_on "go" => :build
 
+  conflicts_with "ghostscript", because: "both install `gs` binaries"
```

Comment:
singular / plural inconsistenct, also in other formulae
```suggestion
  conflicts_with "ghostscript", because: "both install `gs` binary"
```